MembraneRTC/Sources/MembraneRTC/PeerConnectionManager.swift

```diff
@@ -274,6 +276,55 @@ internal class PeerConnectionManager: NSObject, RTCPeerConnectionDelegate {
         sender.parameters = params
     }
 
+    private func extractRelevantStats(rp: RTCStatisticsReport) {
+        rp.statistics.forEach { it1 in
+            let it = it1.value
+            if it.type == "outbound-rtp" {
+                let duration = it.values["qualityLimitationDurations"] as? [String: Double]
+                let qualityLimitation: QualityLimitationDurations = QualityLimitationDurations(
+                    bandwidth: duration?["bandwidth"] ?? 0.0,
+                    cpu: duration?["cpu"] ?? 0.0, none: duration?["none"] ?? 0.0, other: duration?["other"] ?? 0.0)
+                let tmp = RTCOutboundStats(
+                    kind: it.values["kind"] as? String ?? "",
+                    rid: it.values["rid"] as? String ?? "",
+                    bytesSent: it.values["bytesSent"] as? Int ?? 0,
```

Comment:
it's int? not any big number? or `NSDecimalNumber`?

Reply:
Int on a modern IPhone has 63 bit precision, which should be more than enough since when changed from bytes it would mean that the peer must have had used several petabytes to exceed that, and backed would prob crash first :P. I changed it to be UInt to get that additional bit of precision, but I think it is irrelevant.